test/fixtures/output/connect/custom-plugin/update.golden

```diff
@@ -0,0 +1,5 @@
+Updated custom connector plugin "ccp-123456".
```

Comment:
Do we want to remove the update message? How will the customers know the update succeeded?

Reply:
They should see their changes in the output underneath, no?

Reply:
They're just seeing the id and the name. If they had updated the description they wouldn't see it.

Reply:
I was thinking they would have to do describe if they want to see other fields. It might be better to remove the table actually to keep it consistent with other connect commands

Reply:
How about we add the connector description to the output table so they'll see it if it's updated?

Reply:
I can add all the fields since they can all be updated besides class and type but then it is not consistent with how connector update is. Is that okay given this command is under connect?

Reply:
@brianstrauch I removed the table in the latest pr to match connect if you think we should show all updatable properties I can put the table back. Let me know!

Reply:
Let's stay consistent here. Maybe in the next major version update we can switch to table output if that's more desirable.